kuka_sunrise_fri_driver/src/hardware_interface.cpp

```diff
@@ -195,15 +197,20 @@ CallbackReturn KukaFRIHardwareInterface::on_activate(const rclcpp_lifecycle::Sta
       fri_connection_->setClientCommandMode(ClientCommandModeID::POSITION_COMMAND_MODE);
       break;
     case kuka_drivers_core::ControlMode::JOINT_IMPEDANCE_CONTROL:
-      fri_connection_->setJointImpedanceControlMode(hw_stiffness_commands_, hw_damping_commands_);
+      fri_connection_->setJointImpedanceControlMode(
+        hw_joint_stiffness_commands_, hw_joint_damping_commands_);
       fri_connection_->setClientCommandMode(ClientCommandModeID::POSITION_COMMAND_MODE);
       break;
     case kuka_drivers_core::ControlMode::JOINT_TORQUE_CONTROL:
       fri_connection_->setJointImpedanceControlMode(
         std::vector<double>(DOF, 0.0), std::vector<double>(DOF, 0.0));
       fri_connection_->setClientCommandMode(ClientCommandModeID::TORQUE_COMMAND_MODE);
       break;
-
+    case kuka_drivers_core::ControlMode::WRENCH_CONTROL:
+      fri_connection_->setCartesianImpedanceControlMode(
+        std::vector<double>(6, 0.0), std::vector<double>(6, 0.1));
```

Comment:
why is damping 0.1?

Reply:
Because that is the lowest possible value allowed in cartesian impedance mode.

Reply:
okay, hopefully this does not influence the movement too much